WordPress/src/main/java/org/wordpress/android/ui/posts/mediauploadcompletionprocessors/BlockProcessor.java

```diff
@@ -131,7 +131,7 @@ String processBlock(String block) {
         return processBlock(block, false);
     }
 
-     final void addIdPropertySafely(JsonObject jsonAttributes, String idName, String remoteId) {
+     final void addIntPropertySafely(JsonObject jsonAttributes, String propertyName, String value) {
         try {
             jsonAttributes.addProperty(idName, Integer.parseInt(remoteId));
```

Comment:
@pantstamp The changes I suggested https://github.com/wordpress-mobile/WordPress-Android/pull/20495#discussion_r1530647905 weren't meant to be committed directly as it requires also updating this line with the function arguments (`propertyName` and `value`). Sorry because I didn't specify this clearly 😅 .